src/routes/(console)/organization-[organization]/budgetLimitAlert.svelte

```diff
@@ -6,6 +6,9 @@
     import { HeaderAlert } from '$lib/layout';
     import { hideBillingHeaderRoutes, readOnly, showBudgetAlert } from '$lib/stores/billing';
     import { base } from '$app/paths';
+
+    $: redirectUrl = `${base}/organization-${$organization.$id}/billing#update-budget`;
+    $: isSameUrl = $page.url.pathname === new URL(redirectUrl, window.location.origin).pathname;
```

Comment:
Does adding a searchParam to the URL trigger the scroll again? If it's the same URL you could add `?seach=true` and that should trigger the scroll again

This way we don't have to rely on custom code in the target  component

(still not quite sure why it doesn't already work out of the box, when we have more time let's investigate )